runtime/colors/README.txt

```diff
@@ -59,20 +63,20 @@ autocommand event.  For example:
 	augroup END
 
 To customize a colorscheme use another name, e.g.  "~/.vim/colors/mine.vim",
-and use ":runtime" to load the original colorscheme:
+and use ":runtime" to load the original colorscheme: >
 
 	" load the "evening" colorscheme
 	runtime colors/evening.vim
 	" change the color of statements
 	hi Statement ctermfg=Blue guifg=Blue
 
-To see which highlight group is used where, see ":help highlight-groups" and
-":help group-name".
+To see which highlight group is used where, see `:help highlight-groups` and
+`:help group-name`.
```

Comment:
For proper highlighting, the space after the closing backtick is required (as the thing inside the backticks already has a space; yes it's weird)

```suggestion
`:help group-name` .
```